Clients/Xamarin.Interactive.Client.Mac/OutlineView/CollectionOutlineView.cs

```diff
@@ -121,8 +121,14 @@ public CollectionOutlineViewDataSource.NodeProxy GetNodeProxy (nint row)
             get { return GetNodeProxy (SelectedRow); }
             set {
                 nint row = -1;
+                var current = GetNodeProxy (SelectedRow);
```

Comment:
`current` is unused. Did you intend to use it in `DeselectRow`?

Reply:
(This was the only thing I found that might be a functional error)

Reply:
It isn't a functional error because SelectedRow is an nint and remains unchanged but current is indeed unused, removing.